integration/terraform/ec2/README.md

```diff
@@ -112,12 +112,15 @@ require public access.
 
 The security group(s) that the integration tests use should include the following for ingress:
```

Comment:
Personally, I would more favor in setting up the VPC and Security Group inside the terraform instead of setting them manually; moreover, I do not mind if we allows all ingress traffic since its in the VPC.

Reply:
You're welcome to update the EC2 terraform to create the security groups in a separate PR. I'm just trying to keep the document up to date